docs/security.rst

```diff
@@ -56,10 +56,18 @@ semantics as for ``x-tokenInfoFunc``, but the function accepts three
 parameters: username, password and required_scopes. If the security declaration
 of the operation also has an oauth security requirement, required_scopes is
 taken from there, otherwise it's None. This allows authorizing individual
-operations with oauth scope while using basic authentication for
+operations with `oauth scope`_ while using basic authentication for
 authentication.
 
-You can find a `minimal Basic Auth example application`_ in Connexion's "examples" folder.
+You can find a `minimal Basic Auth example application`_ in Connexion's "examples" folder:
+
+
+.. literalinclude:: ../examples/openapi3/basicauth/app.py
```

Comment:
In the example I am wondering if  this line https://github.com/zalando/connexion/blob/bed4b95205205861bd6014282d0f61d50231d3ac/examples/openapi3/basicauth/app.py#L31 makes sense?

Reply:
The example was showing the oauth2 as well, so I fixed that by improving the documentation and linking to the correct example for oauth2, and I removed that dummy_func function as well, as it no longer makes sense.